### PR TITLE
Add Sylo.app latest version

### PR DESCRIPTION
**Pull request by vinkla**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked that the cask was not already refused in [closed issues].

[token reference]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/caskroom/homebrew-cask/pulls
[closed issues]: https://github.com/caskroom/homebrew-cask/issues?q=is%3Aissue+is%3Aclosed

Added Sylo.app http://www.sylomusic.com/

### CODE REVIEW

File: Casks/sylo.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask 'sylo' do
+  version :latest
```
Comment (vitorgalvao) [inline Casks/sylo.rb]: `2.5.0`

File: Casks/sylo.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+  sha256 :no_check
+
+  # s3-us-west-2.amazonaws.com/unision was verified as official when first introduced to the cask
+  url 'https://s3-us-west-2.amazonaws.com/unision/Sylo.zip'
```
Comment (vitorgalvao) [inline Casks/sylo.rb]: Please add an [`appcast`](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/appcast.md):

```
appcast 'http://admin.unisionmusic.com/sylo/update.xml',
         checkpoint: '9e7d6805a219f1ce3476159a9fddf88758d0c2f3577c0284dafd4461148f6e49'
```

You should likely also add `auto_updates true`.

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

File: Casks/sylo.rb (hunk 2 above)
Comment (vinkla) [inline Casks/sylo.rb]: Done.

File: Casks/sylo.rb [hunk 3]
```diff
@@ -0,0 +1,15 @@
+cask 'sylo' do
+  version '2.5.0'
+  sha256 :no_check
```
Comment (vitorgalvao) [inline Casks/sylo.rb]: `sha256` also needs to be set. You can find what it is by running `shasum -a 256` on the downloaded file.

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

File: Casks/sylo.rb [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+
+  # amazonaws.com/unision was verified as official when first introduced to the cask
```
Comment (vitorgalvao) [inline Casks/sylo.rb]: Make it a bit more specific: `s3-us-west-2.amazonaws.com/unision`.

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

File: Casks/sylo.rb (hunk 3 above)
Comment (vinkla) [inline Casks/sylo.rb]: Done.

File: Casks/sylo.rb (hunk 4 above)
Comment (vinkla) [inline Casks/sylo.rb]: Done.

**Review by vitorgalvao — APPROVED** [review]

**Comment by adidalal** [comment]
Thank you for the contribution. It needed some fixes, so they were made in 552ce5573b41969d4e3eac795313f4daf943b970. Your contribution is still included (and still credited to you), with the appropriate modifications. Please feel free to ask about any of the changes.

**Pull request closed without merging**